openassessment/templates/openassessmentblock/grade/oa_grade_complete.html

```diff
@@ -95,104 +100,107 @@ <h4 class="question__title ui-toggle-visibility__control">
                     </article>
 
                     {% if peer_assessments %}
-                        <form id="submission__feedback" class="submission__feedback ui-toggle-visibility step__content__section is--collapsed" method="post">
-                            <h3 class="submission__feedback__title ui-toggle-visibility__control">
-                                <i class="icon fa fa-caret-right" aria-hidden="true"></i>
-                                <span class="submission__feedback__title__copy">{% trans "Provide Feedback on Peer Assessments" %}</span>
-                            </h3>
-
-                            <div class="ui-toggle-visibility__content">
-
-                                <div class="submission__feedback__content {{ has_submitted_feedback|yesno:"is--submitted," }}">
-                                <span class="transition__status is--hidden" aria-hidden="true">
-                                    <span class="wrapper--anim">
-                                        <i class="icon fa fa-spinner fa-spin" aria-hidden="true"></i>
-                                        <span class="copy">{% trans "Submitting Feedback" %}</span>
+                        <form id="submission__feedback" class="submission__feedback step__content__section" method="post">
+                            <li class="is--showing">
```

Comment:
All the steps have an li that encompasses the step, usually called something like openassessment__step-name. I gave that li an is--showing class because it was more straightforward to handle changing the CSS that way (before, the li had an is--collapsed class). But for the ui-slidable components that aren't a step (like the button to collapse/expand rubric questions within the grading steps), there isn't any encompassing li class that would hold the is--showing. I wasn't sure if there was a cleaner way to do this, but I tried several different ways and this seemed to be the most straightforward one.